script.js

```diff
@@ -1,4 +1,64 @@
+/*Basic Scissors Paper Stone
+
+Create a basic version of Scissors Paper Stone where the user inputs one of "scissors", "paper", or "stone", the program internally randomly chooses scissors, paper, or stone, and the program outputs whether the user won, the program won, or it's a draw.
+
+Rules: scissors beats paper, paper beats stone, and stone beats scissors. If both parties choose the same object, it's a draw.
+
+Input Validation
+
+Sometimes the user types bad and types something other than "scissors", "paper", or "stone" during gameplay. Add input validation to kindly let the user know that there are only 3 input options, and ask them to try again.*/
+
+/*pseudo code for scissors paper stone
```

Comment:
This is excellent pseudo code and you clearly communicate what You are going to do here.